### PR TITLE
Update concurrency settings for Integration tests

### PR DESCRIPTION
**Pull request by viniciusdc**
## Reference Issues or PRs
#2386 

## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features not to work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [ ] Did you test the pull request locally?
- [ ] Did you add new tests?

### CODE REVIEW

**Comment by viniciusdc** [comment]
ref. for working config https://github.com/nebari-dev/nebari/actions/runs/8651753113

File: .github/workflows/test_local_integration.yaml [hunk 1]
```diff
@@ -34,14 +34,16 @@ on:
         required: true
         type: string
 
+concurrency:
```
Comment (aktech) [inline .github/workflows/test_local_integration.yaml:37]: Can you add a line here to explain what's the expected behaviour?

Comment (viniciusdc) [inline .github/workflows/test_local_integration.yaml:37]:
```suggestion
# When the cancel-in-progress: true option is specified, any concurrent jobs or workflows using the same
# concurrency group will cancel both the pending and currently running jobs or workflows. This allows only
# one job or workflow in the concurrency group to be in progress at a time.
concurrency:
```

Comment (viniciusdc) [reply .github/workflows/test_local_integration.yaml:40]: @aktech let me know if this helps

**Pull request merged**